modAionImpl/src/org/aion/zero/impl/config/CfgAion.java

```diff
@@ -43,6 +43,12 @@
  */
 public final class CfgAion extends Cfg {
 
+    private static String NETWORK = "mainnet";
```

Comment:
this gets modified by "-n" option in Cli.java (keeps track of current network) and is shown beneath the AION logo when the kernel starts

Reply:
Hmm, but call(String[], Cfg) in Cli.java -- it takes Cfg as an argument.  Couldn't NETWORK, CONF_FILE_PATH, GENESIS_FILE_PATH, just be instsance (non-static) variables, and then CfgAion can have setNetwork, setConfFilePath, setGenesisFilePath methods that Cli uses to set those values?

I know it can be hard to make things non-static is there's a bunch of other static code depending on it, but that does not seem to be the case here.